packages/docs/pages/reference/elements/p.mdx

```diff
@@ -29,4 +30,26 @@ This element can appear as an immediate child of the following elements.
 
 ## Examples
 
+```ptx-example
+<FRAGMENT parents="subsection section">
+    <p>
+        Suppose <m>f</m> were a function on the natural numbers.
+        That would make some of us <em>very</em> interested in it.
+    </p>
+</FRAGMENT>
+```
+
```

Comment:
Maybe a description before this example of the fact that <p>'s are most often the immediate children of other block-type elements.